app.py

```diff
@@ -0,0 +1,40 @@
+
+from opentok import OpenTok
+from flask import Flask, request
+app = Flask(__name__)
+app.config.from_pyfile('.env')
+
+opentok = OpenTok(app.config.get('OPENTOK_API_KEY'), app.config.get('OPENTOK_API_SECRET'))
+
+
+@app.route('/')
+def index():
+    return "Success! Endpoints available: /session."
+
+
+@app.route('/session', methods=['POST'])
+def session():
+    # return "Success! Endpoints available: /session."
```

Comment:
I believe this is an extraneous comment meant for the index route.

Reply:
oops